docs/rules/functional-interface.md

```diff
@@ -88,6 +88,53 @@ AFunctionalInterface foo = (int a1) -> {
 };
 ```
 
+### Bytecode JDK 1.8 
```

Comment:
I really like the bytecode examples. Coming from someone who does not usually look at bytecode, I would like to see a bit of an explanation though. The __Post__ code has more lines in total. Is it compiled into two files?